restructure quota mgmt docs (#5181)
getsentry/sentry-docs: src/docs/product/accounts/quotas/manage-transaction-quota.mdx

This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.

### How can I see a breakdown of incoming events?

The [Usage Stats](/product/stats/#usage-stats) tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: _accepted_, _dropped_, or _filtered_. Only accepted events affect your quota:

![Usage Stats page showing transactions](usage-stats-transactions.png)

### What are my busiest projects?

The "Project" table in the "Usage Stats" tab of **Stats** breaks down your events by project so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings, where you can manage its [inbound filters](#2-inbound-data-filters).

### Which transactions are consuming my quota?

> 

Author: ```suggestion

<Note>

You can only update the columns of the Results table in **Discover** if your organization is on a Business plan.

</Note>

```

Applying the proper SDK configuration is an iterative and on-going process, but these tips will show you how to best use Sentry's tools to get the most out of your transaction events. These tips are ordered from easiest or least time-consuming to most challenging or potentially time-consuming. The first two options are things you can do in [sentry.io](https://sentry.io), whereas the remaining ones are things you'll do in the SDK.

## Before You Begin: Check Your Quota Usage {#3-event-usage-stats}

You can look at your events in aggregate in the "Usage Stats" tab of **Stats**. This information will help you answer key questions about the breakdown of your incoming events or which projects are consuming your quota. The answers to these questions can help you figure out where you need to do further fine-tuning of your SDK filters and configuration.

> This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.

Author: ```suggestion
This page is accessible to all members of your organization, so Owners in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.
```

### How can I see a breakdown of incoming events?

The [Usage Stats](/product/stats/#usage-stats) tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: _accepted_, _dropped_, or _filtered_. Only accepted events affect your quota:

![Usage Stats page showing transactions](usage-stats-transactions.png)

### What are my busiest projects?

The "Project" table in the "Usage Stats" tab of **Stats** breaks down your events by project so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings, where you can manage its [inbound filters](#2-inbound-data-filters).

### Which transactions are consuming my quota?

<Note>

You can only update the columns of the Results table in **Discover** if your organization is on a Business plan.

</Note>

> You can set up [a query](<https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-title&statsPeriod=14d&yAxis=count()>) in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:

Reviewer: ```suggestion
You can set up [a query](<https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-count&statsPeriod=14d&yAxis=count()>) in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:
```

![Busiest Projects](manage-trans-quota-01.png)

> Once the changes are applied, sort the "Results" table by the "COUNT" column to display your busiest transactions:

Reviewer: ```suggestion
Once the changes are applied, sort the "Results" table by the "COUNT()" column to display your busiest transactions:
```

This page is accessible to all members of your organization, so a Billing or Owner member in your Sentry org can share this page with the developers directly responsible for a given project. Also, you can come back to this page to check if the changes you've made are having the desired effect.

### How can I see a breakdown of incoming events?

The [Usage Stats](/product/stats/#usage-stats) tab displays details about the total amount of data Sentry has received across your entire organization for up to 90 days. The page breaks down the events (by project) into three categories: _accepted_, _dropped_, or _filtered_. Only accepted events affect your quota:

![Usage Stats page showing transactions](usage-stats-transactions.png)

### What are my busiest projects?

The "Project" table in the "Usage Stats" tab of **Stats** breaks down your events by project so you can see which ones are consuming your quota. Clicking on the settings icon next to a project name in the table will open the project's settings, where you can manage its [inbound filters](#2-inbound-data-filters).

### Which transactions are consuming my quota?

> You can set up a query in **Discover** to see which transactions you're receiving the most and if you want to adjust that. When you're building the query, search for `event.type:transaction` and then set the columns to `title`, `project`, and `count`, as shown below:

Reviewer 2: Is it worth setting up a warning here to indicate that this search is only possible on Business plan? You could also create this query and use org redirects so that customers could easily navigate right to the page with the query preselected like so:

https://sentry.io/orgredirect/organizations/:orgslug/discover/results/?field=title&field=project&field=count()&name=All+Events&query=event.type%3Atransaction&sort=-title&statsPeriod=14d&yAxis=count()

When you exceed your quota threshold, the server will respond with a 429 HTTP status code, which communicates to SDKs and clients to stop sending events. This status code comes with a `Retry-After` header that indicates the time for which this rate limit is active. However, clients are not supposed to retry events, but instead drop events until the rate limit has expired, to prevent queue backlogs. Note, that since event ingestion and rate limiting happen asynchronously, the 429 HTTP status code is always slightly delayed.

To increase your quota, go to **Settings > Subscription** and click the "Manage Subscription" button to access your subscription options. When you increase your quota, the change goes into effect immediately.

If you're on a Developer plan and want to increase your quota, you'll need to upgrade to a Team or Business plan. On these plans you can prepay for more event volume and purchase on-demand volume, as needed. Learn about Sentry's plans on our [pricing page](https://sentry.io/pricing/).

> <Note>

Reviewer 2: Should this be a warning instead of a note? This causes a fair bit of pain when it does happen.

> <Note>

Author: ```suggestion
<Alert>
```

If you upgrade from Team to Business mid-billing period, your on-demand pricing changes retroactively.

> </Note>

Author: ```suggestion
</Alert>
```
